forecasters/animalia/tests/testthat/test-population.R

```diff
@@ -0,0 +1,109 @@
+library(covidcast)
+library(dplyr)
+
+state_geo_values <- c("ak", "al", "az")
+state_population <- c(731545, 4903185, 7278717)
+county_geo_values <- c("06037", "17031", "49035")
+county_population <- c(10039107, 5150233, 1160437)
```

Comment:
unfortunately, I'm going to ask you to revise all of this test.

Testing with data out of the api is fairly unstable and results in lots of problems. Better is to use the mocking functions. Though here, I don't even think you need to do that. I would create some fake data in place of `state_df_list` and `county_df_list`. Here's [an example](https://github.com/cmu-delphi/covidcast/blob/a15fd603558a2288dd7cbebb9392436f560fd2c6/R-packages/evalcast/tests/testthat/test-evaluate_predictions.R#L4).

Reply:
Revised.